Sources/UInt128.swift

```diff
@@ -147,18 +115,9 @@ public struct UInt128 {
             throw UInt128Errors.emptyString
         }
         // Internal variables.
-        let radix: UInt8
+        let radix: UInt8 = string.radix
```

Comment:
👎

As I mention in the UInt8 extension you have at the bottom of this file, I do not feel comfortable adding properties to built in types within a framework like this. Within your own personal application, it makes a ton of sense, but not in a low-level Framework.